examples/inference/run.sh

```diff
@@ -0,0 +1,55 @@
+script_dir=$(cd "$(dirname "$0")" && pwd)
+cd "${script_dir}"
+
+
+# 7b, fp16, 2 gpu, 1024, 128
+for BATCH_SIZE in 2 4 8 16; do
+    CUDA_VISIBLE_DEVICES=0,1,2,3 python ./benchmark.py \
```

Comment:
CUDA_VISIBLE_DEVICES shoud not be set in this script, as we don't know which GPU users want to use

Reply:
removed.